docs/user/resources/04-60-gcp-redis-instance.md

```diff
@@ -0,0 +1,85 @@
+# GcpRedisInstance Custom Resource
+The `gcpredisinstance.cloud-resources.kyma-project.io` is a namespace-scoped custom resource (CR).
+It describes the Google Memorystore Redis instance.
+Once the instance is provisioned, inside the same namespace, an "auth" `Secret` with endpoint and credentials is provided alongside it.
+
+Current implementation supports *Basic* and *Standard(without replicas)* tiers, that are explained in detail on the [Google's Memorystore for Redis overview page](https://cloud.google.com/memorystore/docs/redis/memorystore-for-redis-overview).
+
+Redis requires a `/28` ip range.
+To learn more, read [Configure a reserved IP address range](https://cloud.google.com/filestore/docs/creating-instances#configure_a_reserved_ip_address_range).
+Those IP addresses are allocated from the [IpRange CR](./04-10-iprange.md).
+If an IpRange CR is not specified in the GcpNfsVolume, then the default IpRange is used.
+If a default IpRange does not exist, it is automatically created.
+Manually create a non-default IpRange with specified Classless Inter-Domain Routing (CIDR) and use it only in advanced cases of network topology when you want to be in control of the network segments in order to avoid range conflicts with other networks.
+
+When creating GcpRedisInstance, two fields are mandatry: `memorySizeGb`, and `tier`.
+
+Optionally, you can specify the `redisVersion`, `authEnabled`, `transitEncryption`, `redisConfigs`, and `maintenancePolicy`.
+
+By default, the created secret that holds credentials has the same name as the GcpRedisInstance, unless specified otherwise.
```

Comment:
By secret, do you mean like Kubernetes Secret? If so, please use uppercase Secret here and in other sections of the document.

Reply:
Changed to uppercase